modules/RedisTimeSeriesEdge/Dockerfile.amd64

```diff
@@ -1,11 +1,14 @@
 FROM mcr.microsoft.com/dotnet/core/sdk:3.1-buster AS build-env
+ARG BUILD_VERSION
+ENV BUILD_VERSION=$BUILD_VERSION
+
 WORKDIR /app
 
 COPY *.csproj ./
 RUN dotnet restore
 
 COPY . ./
-RUN dotnet publish -c Release -o out
+RUN dotnet publish -c Release -o out /p:AssemblyVersion=$BUILD_VERSION /p:FileVersion=$BUILD_VERSION /p:Version=$BUILD_VERSION

```

Comment:
out of curiosity, why we need these 3 versions: `AssemblyVersion`, `FileVersion`, `Version`? Wouldn't be `AssemblyVersion` enough?

Reply:
Good catch! The .NET ecosystem loves versions so much there are so many to choose from :) I refreshed my understanding from [this blogpost](https://andrewlock.net/version-vs-versionsuffix-vs-packageversion-what-do-they-all-mean/) and learned that `Version` is enough to be set. `AssemblyVersion` and `FileVersion` both default to the `Version` value.